packages/react-devtools-shared/src/devtools/store.js

```diff
@@ -448,17 +453,18 @@ export default class Store extends EventEmitter<{
   // This build of DevTools supports the legacy profiler.
   // This is a static flag, controled by the Store config.
   get supportsProfiling(): boolean {
-    return this._supportsProfiling;
+    return this._supportsProfiling || this._supportsReloadAndProfileOverride;
```

Comment:
Would `_supportsReloadAndProfileOverride` ever be true but `_supportsProfiling` is false?

Reply:
Yes, I believe when locally testing in react native `_supportsProfiling` came out as false. Perhaps this is because I was using the standalone profiler, which does not pass the `supportsProfiling` configuration flag.

But TBH, there's a lot going on in this code and I'm not sure.